Color changing/post.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
```

Comment:
What is this page for?

Reply:
This is to showcase the projects from portfolio.